tests/cluster/tests/14-consistency-check.tcl

```diff
@@ -105,8 +112,15 @@ proc test_slave_load_expired_keys {aof} {
         # start the replica again (loading an RDB or AOF file)
         restart_instance redis $replica_id
 
+        # Replica may start a full sync after restart, trying in a loop to avoid
+        # -LOADING reply in that case.
+        wait_for_condition 1000 50 {
+            [catch {set replica_dbsize_3 [R $replica_id dbsize]} e] == 0
```

Comment:
@sundb I got a feeling that this test wants to check dbsize even before replication (just after restart). So, I added catch + loop, we'll try in a loop. Most of the time, successful execution will be before replication. On a slow CI, it might be after replication. In both cases, it should give us the correct result.